EventMixing/src/Mu2eProductMixer.cc

```diff
@@ -366,11 +366,11 @@ namespace mu2e {
                                           StrawGasStepCollection& out,
                                           art::PtrRemapper const& remap)
   {
-    std::vector<StrawGasStepCollection::size_type> stepOffsets;
-    art::flattenCollections(in, out, stepOffsets);
+    //std::vector<StrawGasStepCollection::size_type> stepOffsets;
```

Comment:
I was away on vacation and just saw this now.  With few exceptions , we request that people not level commented out code in files.  The main exceptions is code under development.  It can be Ok to comment out debugging code  but it's generally preferred to wrap debugging code in a verbosity/debug_level  test.

Please keep this in mind moving forward and fix as targets of opportunity.